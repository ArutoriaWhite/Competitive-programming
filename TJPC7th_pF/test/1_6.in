67415764 2
29772593 0
47011776 1
