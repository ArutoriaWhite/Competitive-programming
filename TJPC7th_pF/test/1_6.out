29772593 17239183
